src/templates/static/get_started/index.jsx

```diff
@@ -310,6 +375,7 @@ const Index = () => (
                             <span>{it.L('2 * (6,000 – 5,200) = $1600')}</span>
                         </div>
                         <p>{it.L('Lookbacks options are currently only available for [_1]Volatility Indices[_2].', `<a href=${it.url_for('get-started/binary-options')}#range-of-markets>`, '</a>')}</p>
+                        <VolatilityIndicesTable />
```

Comment:
we also need the sentence `Return to player (RTP) % for lookbacks for a multiplier of 1 and 1m duration is around 87% on average` above the table, as per the card